lib/src/screens/withdraw/withdraw_screen.dart

```diff
@@ -153,6 +153,11 @@ class _WithdrawScreenState extends ConsumerState<WithdrawScreen> {
                       onChanged: (value) {
                         setState(() {
                           sourceCurrency = value;
```

Comment:
Ya que modificaste esto, podríamos aprovechar y mover esto a una función para mejor legibilidad y quitar lógica del build

```suggestion
                          onChanged: updateTargetAmount(value)
```